### PR TITLE
feat: github action to run pipeline

### PR DESCRIPTION
**Pull request by tpluscode**
This PR extends the transformation pipeline and the Job class, and adds a GitHub Action to run the tranformation

### CODE REVIEW

File: fuseki/sample-ubd.trig [hunk 1]
```diff
@@ -326,6 +326,8 @@ graph <project/ubd/csv-mapping/jobs/test-job> {
     cc:tables <cube-project/ubd/csv-mapping/tables> ;
     schema:label "cli-test" ;
     cc:cubeGraph <cube-project/ubd/cube-data> ;
-    dcterms:created "2020-10-29T12:01:54"^^xsd:dateTime
+    dcterms:created "2020-10-29T12:01:54"^^xsd:dateTime ;
+    rdfs:seeAlso <https://github.com/zazuko/cube-creator/actions/runs/345020887> ;
+    schema:actionStatus schema:PotentialActionStatus ;
```
Comment (tpluscode) [inline fuseki/sample-ubd.trig:331]: I propose to extend `cc:Job` from [`schema:Action`](http://schema.org/Action) and use its properties to keep track of the job's state

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -35,6 +35,10 @@ jobs:
       - name: logs on fail
         if: ${{ failure() }}
         run: lando logs -s core
+      - name: Codecov
+        uses: codecov/codecov-action@v1.0.5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```
Comment (lucafurrer) [inline .github/workflows/ci.yaml:41]: How comes that this is a change?

File: apis/core/lib/storage/s3.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import aws from 'aws-sdk'
-import { isReadable } from 'isstream'
+import { readable } from 'is-stream'
```
Comment (lucafurrer) [inline apis/core/lib/storage/s3.ts:2]: What is the difference between those two libraries?

**Review by lucafurrer — APPROVED** [review]
I have not run the code, but it looks fine :+1:

File: apis/core/lib/storage/s3.ts (hunk 3 above)
Comment (tpluscode) [reply apis/core/lib/storage/s3.ts:2]: Not much I guess other than the new is only 2 years old while the other was last released 6 yrs ago :)

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (tpluscode) [reply .github/workflows/ci.yaml:41]: Missed that in #152 and now it was bringing the % down

**Pull request merged**